samples/todo-app/database-api/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim AS base
+WORKDIR /app
+EXPOSE 80
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-buster AS build
+WORKDIR /src
+COPY ["databaseApi.csproj", ""]
+RUN dotnet restore "./databaseApi.csproj"
+COPY . .
+WORKDIR "/src/."
+RUN dotnet build "databaseApi.csproj" -c Release -o /app/build
+
+FROM build AS publish
```

Comment:
why two different steps? dotnet publish builds the projects also?

Reply:
This is the Dockerfile generated by default by Visual Studio for new projects. Unless changing this would have an impact for our customers, I'd rather keep it as generated.